model/riscv_pmp_regs.sail

```diff
@@ -216,3 +244,15 @@ function pmpWriteAddr(locked: bool, tor_locked: bool, reg: xlenbits, v: xlenbits
   if   sizeof(xlen) == 32
   then { if (locked | tor_locked) then reg else v }
   else { if (locked | tor_locked) then reg else EXTZ(v[53..0]) }
+
+function mseccfgWrite(reg: xlenbits, v: xlenbits) -> xlenbits =   /* to set RLB, need to check PMPCFG_L */
```

Comment:
Take the Mseccfg_ent?

Reply:
Corrected the function based on this "Mseccfg_ent" now